reader.py

```
@@ -133,6 +212,281 @@ def next_image_data(self) -> bytes:
+
+class ReadingSequence(QObject):
+    """
+    A class to manage the order of comic display images.
+
+    The order depends on the reading type.
+    """
+
+    front_cover_reached = Signal()
+    back_cover_reached = Signal()
+
+    def __init__(self, comic: Comic):
```

Ruff (0.15.21)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)